src/appkit.cs

```diff
@@ -18357,6 +18357,7 @@ partial interface NSTypesetter {
 	delegate void NSWindowTrackEventsMatchingCompletionHandler (NSEvent evt, ref bool stop);
 	
 	[BaseType (typeof (NSResponder), Delegates=new string [] { "Delegate" }, Events=new Type [] { typeof (NSWindowDelegate)})]
+	[DisableDefaultCtor] // Handled in NSWindow.cs
```

Comment:
IIRC, it's tacky a bit, you can use `[DisableDefaultCtor]` and still do a `IntPtr Constructor();` which could have the `[PostSnippet]` too (and keep the generated code).